packages/authgear-react-native/index.d.ts

```diff
@@ -928,56 +933,94 @@ export declare class ReactNativeContainer {
+   * After authentication, the webview will be closed and the user is logged in.
+   *
+   * You can refer to {@link AuthenticateOptions} for more customization.
    *
    * @public
    */
   authenticate(options: AuthenticateOptions): Promise<AuthenticateResult>;
   /**
+   * changePassword() opens the settings page in a webview for the user to change their password.
+   * After changing the password, the webview will be closed.
+   *
+   * You can refer to {@link SettingsActionOptions} for more customization.
+   *
    * @public
    */
   changePassword(options: SettingsActionOptions): Promise<void>;
   /**
+   * deleteAccount() opens the settings page in a webview for the user to delete their account.
+   * After deletion, the webview will be closed and the user is logged out.
+   *
+   * You can refer to {@link SettingsActionOptions} for more customization.
+   *
    * @public
    */
   deleteAccount(options: SettingsActionOptions): Promise<void>;
   /**
-   * Reauthenticate the end user via biometric or the web.
+   * reauthenticate() starts the reauthentication process via biometric or in the webview.
    *
    * If biometricOptions is given, biometric is used when possible.
    *
+   * You can refer to {@link ReauthenticateOptions} and {@link BiometricOptions} for more customization.
+   *
    * @public
    */
   reauthenticate(
     options: ReauthenticateOptions,
     biometricOptions?: BiometricOptions
   ): Promise<ReauthenticateResult>;
+  /**
+   * Open Authgear pages. Currently only settings pages are available.
+   *
+   * You can refer to {@link SettingOptions} for more customization.
+   *
+   * @public
+   */
   open(page: Page, options?: SettingOptions): Promise<void>;
   /**
-   * Logout.
+   * logout() starts the logout process in the background.
+   * After logging out, the access token will be revoked and cleared from the token storage.
    *
    * @remarks
    * If `force` parameter is set to `true`, all potential errors (e.g. network
    * error) would be ignored.
    *
    * @param options - Logout options
+   *
+   * @public
    */
   logout(options?: { force?: boolean }): Promise<void>;
   /**
-   * Authenticate as an anonymous user.
+   * authenticateAnonymously() starts the authentication process as an anonymous user.
+   * You may first enable Anonymous Users in Authgear Portal (Your project \> Authentication \> Anonymous Users \> Enable anonymous users).
+   *
+   * @public
    */
   authenticateAnonymously(): Promise<AuthenticateResult>;
   /**
-   * Open promote anonymous user page
+   * promoteAnonymousUser() opens the anonymous user promotion page in the webview and the user has to authenticate.
+   * The flow is similar to authenticate(), the webview will be closed and the user is logged in after authentication.
+   * After promotion from anonymous user, biometric authentication will be disabled for the new user
+   * and the may need to enable it manually again.

```

Comment:
The 

> After promotion from anonymous user, biometric authentication will be disabled for the new user and the may need to enable it manually again.

is actually applicable to `authenticate()` as well. If you look at the source code, you will see

Both of them disable biometric if the operation finishes.

So you can use the same description to convey this.

> After this method resolves, biometric authentication is disabled. Call enableBiometric() again to enable biometric authentication for the new signed in user.

Reply:
ok!